### DISCUSSION TITLE
Save my environments in the cloud

### DISCUSSION THREAD

**Discussion by crisdcantillo** [mockoon/mockoon#381] [Ideas] [answered]
Hi there!

I recently found this app and I'm so happy with it. The only thing I'm worried about is that I cannot save all my endpoints in some place in the cloud. I think that this could be an amazing feature!

**Comment by 255kb** [comment, marked as answer]
Hi! 
Thank you for the suggestion. This is definitely on the long term roadmap. 
In fact, it may be part next year of a broader plan to add paid services (online sync). The app would stay free and open-source of course, but such services that incur costs (servers, etc) could be a paid option. 
In the meantime, you can "easily" set up a git repository in the data folder used by Mockoon ("Help" menu -> "Show app data folder", then navigate to `./storage`). This should be even easier after issue #16 is resolved. It will allows you to change the data folder for each environment, and thus select a Google Drive folder or an existing git repo.

**Reply by crisdcantillo** [reply]
Thanks a lot for the answer! A paid version with that function will be amazing! And of course I will try what you tell me here.

Keep working on man, you have a really cool project in your hands!

**Reply by 255kb** [reply]
Thanks! We will do our best 😸
